src/api.js

```diff
@@ -286,16 +295,17 @@ builder.declare({
   }
 
   // Not all webhook payloads include an e-mail for the user who triggered an event
-  let headUser = msg.details['event.head.user.id'].toString();
+  let headUser = msg.body.sender.id.toString();
   let userDetails = (await instGithub.users.getById({id: headUser})).data;
-  msg.details['event.head.user.email'] = this.ajv.validate({type: 'string', format: 'email'}, userDetails.email) ?
-    userDetails.email :
-    msg.details['event.head.user.login'].replace(/\[bot\]$/, '') + '@users.noreply.github.com';
+  msg.details['event.head.user.email'] = this.ajv.validate({type: 'string', format: 'email'}, userDetails.email)
+    ? userDetails.email
+    : msg.body.sender.login.replace(/\[bot\]$/, '') + '@users.noreply.github.com';
```

Comment:
This bit looks unrelated to checks?

Reply:
so, since most of the syntax changes in this PR are left for a "good first bug", and there's no eslint rule that enforces ternary operator formatting, I'll be leaving this particular change in this PR